scripts/ci-frontend-metrics.sh

```diff
@@ -4,7 +4,7 @@ echo -e "Collecting code stats (typescript errors & more)"
 
 ERROR_COUNT_LIMIT=6850
 DIRECTIVES_LIMIT=173
-CONTROLLERS_LIMIT=136
+CONTROLLERS_LIMIT=137
```

Comment:
Thats sad but it is what it is.

Reply:
it's a false positive, it's due to this line
 https://github.com/grafana/grafana/pull/16364/files#diff-7d38adb6eb3fcc661871ef8d0968ffcfR170